exercise/api/custom_serializers.py

```diff
@@ -49,10 +49,20 @@ def submission_url(submission_id):
                 'submission_id': submission_id
             }, request=request)
 
+        def submission_obj(submmission_cached):
+            id_ = submmission_cached['id']
+            return {
+                'id': id_,
+                'url': submission_url(id_),
+                'submission_time': submmission_cached['date'],
+                'grade': submmission_cached['points'], # NOTE: model has this in field `grade`, TODO rename
```

Comment:
Many exercise related API endpoints use the word `points` for the current best points of the student, but the submission detail endpoint uses `grade`. Then it makes sense to use `grade` here as well since this object refers directly to the submission detail URL. The comment in the code could be removed, or was there something else to consider here?

Reply:
@raphendyr

Reply:
Pointed out that why it's so and to mention that someone should probably do something about it some day. It's totally possible to add also variable `points` (in all locations) and then remove `grade` when moving to `v3` API.

Reply:
I think the comment is misleading. If `grade` is changed here to `points`, the change should be done in a few other places too, not just here. Even if `points` is the future choice, it would be confusing if `points` is used here right now and `grade` in the submission detail endpoint.

Reply:
The comment should be written better, for sure. And yes, the key should be now grade, but it would be fine all these locations with a comment to do something about it. Or one could just write an issue about it and leave out the comment (maybe).

Reply:
I removed the comment, for now, I didn't come up with a good comment for this, might be easier to write an issue. I can add a comment here if someone has a good suggestion